plugins/flytekit-envd/flytekitplugins/envd/image_builder.py

```diff
@@ -91,7 +91,7 @@ def create_envd_config(image_spec: ImageSpec) -> str:
     run_commands = _create_str_from_package_list(image_spec.commands)
     conda_channels = _create_str_from_package_list(image_spec.conda_channels)
     apt_packages = _create_str_from_package_list(image_spec.apt_packages)
-    env = {"PYTHONPATH": "/root", _F_IMG_ID: image_spec.image_name()}
+    env = {"PYTHONPATH": "/root:.", _F_IMG_ID: image_spec.image_name()}
```

Comment:
The "." is easy to miss. I think using `PWD` works too?

```suggestion
    env = {"PYTHONPATH": "/root:$PWD", _F_IMG_ID: image_spec.image_name()}
```

Reply:
you don't need `.` you just need the `:`

Reply:
how does this work? I was under the impression that the current working directory was already part of `sys.path` when the python interpreter starts.

Reply:
> For custom base images, should ImageSpec change the WORKDIR to /root?

we don't need to. It should use the workdir in the base image.